src/ZiggyCreatures.FusionCache/FusionCacheEntryOptions.cs

```diff
@@ -63,6 +65,16 @@ public FusionCacheEntryOptions(TimeSpan? duration = null)
 	/// </summary>
 	public TimeSpan Duration { get; set; }
 
+	/// <summary>
+	///  Gets the Microsoft.Extensions.Primitives.IChangeToken instances which cause the cache entry to expire.
+	/// <br/><br/>	
+	/// Cancelling th source forces an instant eviction from cache and calls FusionCache.Remove on the key to remove from distributed cache and send backplane anotification
+	/// <br/><br/>	
+	/// If set on DefaultEntryOptions, reinstantiate this property every time the CancellationTokenSource is Cancelled.
```

Comment:
Recommending the use of `see` with `DefaultEntryOptions` and `CancellationTokenSource` here. Additionally, not sure if `Cancelled` is supposed to be capitalized.